delfin/drivers/hitachi/vsp/consts.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+SOCKET_TIMEOUT = 52
+ERROR_SESSION_INVALID_CODE = 403
+ERROR_SESSION_IS_BEING_USED_CODE = 409
+BLOCK_SIZE = 512
+
+VSP_FXXX_GXXX_SERIES = ('VSP G350', 'VSP G370', 'VSP G700', 'VSP G900',
```

Comment:
VSP_F_G_SERIES is better for me.

Reply:
there also have like F200,G200  they are different